hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -462,7 +462,7 @@ objects:
           sourceType: linux_audit
           whiteList: \.log$
         - index: rh_osd_suricata
-          path: /host/var/log/suricata/*.log
+          path: /host/var/log/openshift_managed_suricata.log
```

Comment:
@rhdedgar - I believe you said suricata is currently logging to `/var/log/suricata.log` on the host, right?

Reply:
+1 for this should be /var/log/suricata/*.log I think. Currently suricata supports a log directory configuration: https://suricata.readthedocs.io/en/suricata-6.0.0/configuration/suricata-yaml.html#default-logging-directory.

The existing container has it setup for `/host/var/log` which maps to `/var/log/` on the host.

The four important files are:
* `suricata.log` - status/error messages about suricata itself
* `stats.log` - network stats
* `fast.log` - suspicious activity found by suricata
* `eve.json` - Basically stats.log + fast.log in JSON format (maybe unnecessary, or maybe we prefer this since Splunk does JSON parsing?)

Reply:
Can confirm that the host's /var/log directory is used by suricata: https://github.com/rhdedgar/suricata/blob/master/Containerfile#L9.